src/app/components/Signup/Signup.tsx

```
@@ -0,0 +1,351 @@
+				email: id,
+				password: password,
+				name: name,
+				companyName: companyName,
+			});
+			setErrorId('');
+			setErrorPassword('');
+			setErrorCompanyName('');
+			setErrorPassword('');
+			setErrorPasswordConfirm('');
+			router.back();
+		} catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)